streams/src/test/java/org/apache/kafka/streams/tests/SmokeTestDriver.java

```diff
@@ -106,7 +106,11 @@ private static class ValueList {
         }
 
         int next() {
-            return (index < values.length) ? values[index++] : -1;
```

Comment:
The test produces too much data, and thus ended up sending `-1` for a lot of records at the end.